db/1.9/modules/ROOT/pages/database-configure-data-types.adoc

```diff
@@ -0,0 +1,78 @@
+= Configure Database Connector Data Types Examples - Mule 4
+
+The following examples shows how to configure Anypoint Connector for Database (Database Connector) data types by using the Parameters Type field to specify the type of each input parameter in the query, and the Column Types field to set custom data types when connected to a particular database provider.
```

Comment:
ditto noun-verb agreement:

examples show
OR
example shows

Reply:
Don't need comma

Reply:
See 1265 edit for questions about capitalization; I'll leave them be in this file.